content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -184,6 +190,13 @@ before proceeding to steps in this section.
 > These operations must be run on the Istio control plane cluster
 to capture the Pilot, Policy, and Statsd Pod IP endpoints.
 
+> As decumented in [istio/istio#4822](https://github.com/istio/istio/issues/4822),
+if any one of the endpoints listed restarts, then the connection from remote
+clusters to that endpoint will be broken. See [Stable access to istio control
+plane services](#stable-access-to-istio-control-plane-services] for details on
```

Comment:
closing ']' when should be a ')'

Reply:
i thought you were removing this upfront link that could cause non-linear reading of the documentation?

Reply:
@sdake - I removed the link and discussion about the issue at the very beginning of the document as we discussed.  I can also remove this one if you wish